addons/account_peppol/wizard/peppol_registration.py

```diff
@@ -293,21 +289,18 @@ def button_register_peppol_participant(self):
         # success
         notifications = {
             'sender': {
-                'title': _('Registered as a sender.'),
                 'message': _('You can now send electronic invoices via Peppol.'),
             },
             'smp_registration': {  # TODO remove in master
-                'title': _('Registered to receive documents via Peppol.'),
-                'message': _('Your registration on Peppol network should be activated within a day. The updated status will be visible in Settings.'),
+                'message': _('Your Peppol registration will be activated soon. You can already send invoices.'),
             },
             'receiver': {
-                'title': _('Registered as a receiver.'),
                 'message': _('You can now send and receive electronic invoices via Peppol'),
             },
         }
         state = self.company_id.account_peppol_proxy_state
         return self._action_send_notification(
-            title=notifications[state]['title'],
+            title=None,
             message=notifications[state]['message'],
         )
```

Comment:
```py
return self._action_send_notification(message=notifications[state]['message'])
```
? :thinking:

Reply:
I named them for clarity in the first place, but both parameters are not optional in the signature...
I can't change the signature in stable so.... beautiful None :face_holding_back_tears:

Reply:
If that's ok for you yes, both parameters are not optional in the signature so it felt more correct to still give the explicit None. I'm fine with both.